packages/earth-admin/src/components/org-switcher/OrgSwitcher.tsx

```diff
@@ -40,6 +40,16 @@ const OrgSwitcher = (props) => {
     dropdownState === 'close' ? setDropdownState('open') : setDropdownState('close');
   };
 
+  const getAvailableOrgs = () => {
+    const specialPermissions = [
+      '*' // super-admin
+    ];
+
+    return Object
+      .keys(permissions)
+      .filter(permission => !specialPermissions.includes(permission));
+  };
+
```

Comment:
- Reuse this function by moving it into utils
- pass permissions as parameter
- add types for parameter and return value 
- add a description comment above the function